File: app/src/main/java/com/overswayit/plesnisavezsrbije/models/Couple.java
package com.overswayit.plesnisavezsrbije.models;

import com.overswayit.tuple.Quadruple;
import com.overswayit.tuple.Triple;

/**
 * Created by lazarristic on 25/02/2019.
 * Copyright (c) 2019 PlesniSavezSrbije. All rights reserved.
 */
public class Couple {
    public String id;
    public String nameMale;
    public String nameFemale;
    public Club club;

    public Triple<AgeCategory, DanceCategory, String> pointListLA;
    public Triple<AgeCategory, DanceCategory, String> pointListST;
    public Quadruple<AgeCategory, DanceCategory, String, String> ratingListLA;
    public Quadruple<AgeCategory, DanceCategory, String, String> ratingListST;
    public Quadruple<AgeCategory, DanceCategory, String, String> ratingListKM;

}

File: app/src/main/java/com/overswayit/plesnisavezsrbije/models/Couple.kt
package com.overswayit.plesnisavezsrbije.models

import androidx.room.ColumnInfo
import androidx.room.Embedded
import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * Created by lazarristic on 25/02/2019.
 * Copyright (c) 2019 PlesniSavezSrbije. All rights reserved.
 */
@Entity
class Couple(@PrimaryKey var id: String,
             @ColumnInfo(name = "name_male") var nameMale: String,
             @ColumnInfo(name = "name_female") var nameFemale: String,
             @Embedded(prefix = "club_") var club: Club)
